components/events/EventCard.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+    <div class="card">
+        <div class="title_block">
+            <div class="title">{{ title }}</div>
+            <div class="tag" :class="[tagColor]">{{ tag }}</div>
+        </div>
+        <p class="card_txt">
+            {{ description }}
+        </p>
+        <div class="card_img">
+            <img :src="imgUrl" alt="imgAlt" />
```

Comment:
The `alt` attribute should use `v-bind`, i.e., `v-bind:alt="imgAlt"` since the `imgAlt` is a variable defined in the `<script>`. However, we often use `v-bind` shorthand, which is denoted as `:alt="imgAlt"` in this case.
```suggestion
            <img :src="imgUrl" :alt="imgAlt" />
```

Reply:
fixed